examples/resources/equinix_fabric_connection/port_to_alibaba.tf

```diff
@@ -0,0 +1,39 @@
+resource "equinix_fabric_connection" "port2alibaba" {
+  name = "ConnectionName"
+  type = "EVPL_VC"
+  notifications {
+    type = "ALL"
+    emails = ["example@equinix.com", "test1@equinix.com"]
+  }
+  bandwidth = 50
+  redundancy { priority = "PRIMARY" }
+  order {
+    purchase_order_number = "1-323929"
+  }
+  a_side {
+    access_point {
+      type = "COLO"
+      port {
+        uuid = "<aside_port_uuid>"
+      }
+      link_protocol {
+        type     = "DOT1Q"
+        vlan_tag = "2019"
+      }
+    }
+  }
+  z_side {
+    access_point {
+      type               = "SP"
+      authentication_key = "<alibaba_account_id>"
+      seller_region      = "us-west-1"
+      profile {
+        type = "L2_PROFILE"
+        uuid = "<service_profile_uuid>"
+      }
+      location {
+        metro_code = "SV"
+      }
+    }
+  }
+}
```

Comment:
I missed this; the instructions only work if you include the alicloud resources in this example. I had assumed you had included them but I was mistaken. 

Please select an option from the following and update the PR to match:
1. You update the instructions to only refer to what needs to be done with just a connection example to Alibaba without Alicloud provider integration (I prefer this to avoid needing to update our Terraform registry if Alicloud updates)
  * If this approach is taken (which I would suggest) you only need to include points 3 and 5 with rewording to make them appropriate for running the example file, accepting from Alibaba Portal, then deleting, and finishing the deletion from Alibaba portal.
3. You include the VBR and supporting information in the example so that the example makes sense

Choice is yours, but it does need to be modified to reflect what is actually in the example.

Let me know if this is confusing and we'll jump on a short call.

Reply:
After processing this, we most definitely want to go with option #1

Reply:
Would also be good to add a trailing newline in the example.tf as well. You can configure your editors to automatically do that for you. I would recommend that as well.

Reply:
@thogarty  - 
Approach 1 - I think points 1, 3 and 5 would be needed. Point 1 can also mention about running the example.

Reply:
You can merge them into 1. Create and Accept connection

Reply:
Okay.👍